site/content/help/early-access/index.md

```diff
@@ -53,7 +53,7 @@ You should now find that you have `early-access` as one of your Active release c
 >}}
 
 #### Step 5
-Now that your account is linked, Copy your login token:
+Now that your account is linked, Copy your login token (highlight & CTRL+C):
```

Comment:
`Copy` shouldn't be capitalized

If you're going to explain how to do something, we should explain it clearly:

`Now that your account is linked, copy your login token by highlighting the text and pressing CTRL + C on your keyboard.`